web/src/containers/activity/All.js

```diff
@@ -17,7 +17,9 @@ const All = ({ add, activities }) => {
         {activities.length} program activities
       </h3>
       {activities.map((_, index) => (
-        <EntryDetails activityIndex={index} />
+        <EntryDetails
+          activityIndex={index}
+          key={index} /> // eslint-disable-line react/no-array-index-key
```

Comment:
> When you don’t have stable IDs for rendered items, you may use the item index as a key as a last resort.

https://reactjs.org/docs/lists-and-keys.html

If there's a better option than using array index as key in this instance, please let me know.

Reply:
Turns out there was a `key` prop on the activity object.